### DISCUSSION TITLE
Why not provide a detached method for event listeners?

### DISCUSSION THREAD

**Discussion by lhjok** [mdbootstrap/TW-Elements#1861] [Chat about dev stuff]
In addition to react and vue, there are many other frameworks. When I use these frameworks, the event listener cannot be separated automatically. I need to manually call the method of separating the event listener.

**Comment by lhjok** [comment]
`Input` also cannot customize event listeners

**Comment by juujisai** [comment]
We do not provide the customization for event listeners. Event listeners for the custom events can be added after a component mounts (vue - onMounted, react - useEffect).

**Reply by lhjok** [reply]
I don't use Vue and React, but use Rust's web framework. After the component is unloaded, the event listener will not be cleared together, so I need to separate the event listener by myself.

**Reply by lhjok** [reply]
If a method can be provided, we can transplant it to any framework at will, and it’s just a matter of little effort, why not support it!

**Reply by juujisai** [reply]
Thanks for the idea, we will keep that in mind

**Reply by lhjok** [reply]
If you are willing to do this, please provide a separate initialization method and event separation method. We will be very grateful!
